scripts/partition_manager/partition_manager.rst

```diff
@@ -38,6 +42,25 @@ Container partitions
 
 The start addresses and sizes of image partitions are used in the preprocessing of the linker script for each image.
 
+RAM partition types
+=====================
+
+Image RAM partitions
+   An image RAM partition consists of all RAM that is not defined as a permanent image RAM partition or placeholder RAM partition.
+   It is the default RAM partition associated with an image.
+   If an image must reserve its RAM area permanently, it must use a permanent image RAM partition, described below.
```

Comment:
We should probably clarify the lifetime of the image RAM partition a bit. Permanently -> at the same time as another image in the same domain is running.

Reply:
Changed to `   If an image must reserve its RAM area permanently (i.e. at the same time as other images are running), it must use a permanent image RAM partition, described below.
`